### ISSUE TITLE
Neuron Relatedness

### ISSUE THREAD

**Issue by imichaelnorris** [openconnectome/Statistical-Connectomics-Class#256]
The paper that I'm loosely basing my project on takes its data from the worm atlas [1].

The data is in an excel spreadsheet for reasons unbeknownst to me.  When I parsed it, each row represented a connection between two neurons, and contained something called the "relatedness" of the two neurons.

The relatedness is defined as the number of cell divisions separating Neuron 1 and Neuron 2.  It's some sort of distance metric.

What is relatedness useful for?

[1] http://www.wormatlas.org/neuronalwiring.html || 2.4 Lineage Analysis

**Comment by indivaux** [comment]
After you posted this I read up on how they do that and it's incredibly awesome! I didn't know they could do that. As far as connectomics is concerned, it might be possible to create some sort of likelihood of connection function based in relatedness. I'm just spitballing here, but for cortical excitatory neurons (which grow next to each other and don't migrate over from other parts of the brain like inhibitory interneurons) you could assign some sort of likelihood that it will connect to the same place as one of it's most related brethren versus the considerably less likely chance that it would connect to the same place as a neuron that it's more distantly related to. However, C. elegans doesn't have a cortex, and I'm not sure how useful this metric would be for organisms that have such a small nervous system.

**Comment by indivaux** [comment]
You could still definitely look at where different neurons connect, keeping their progeny in mind and see if an obvious pattern emerges. However, because of the extensiveness of external factors that prompt neurons to connect in their certain ways, progeny might not play as big of a role as would be useful.

**Comment by imichaelnorris** [comment]
Cool, thanks for the insight.

**Comment by sgomezr** [comment]
Wait, are you using the C elegans connectome for your final project? Were you able to find a good list that separates the neuron list by organism? Because the one that the Worm Atlas providesa neuron has 395 neurons, but that's because it sums the Connectome of the hermaphrodite + the male. They provide a neuron adjacency list with 292 unique values. There are two independent teams trying to map the Male Connectome and the hermaphrodite Connectome by their own, but they state that their results aren't complete. I'm using a partial Connectome for my project. 

Where's your data from? I know that the Central is the WormAtlas, but there's 

1) the general table (which is reportedly the hermaphrodite's Connectome, even though it has more neurons than what it is usually reported for the hermaphrodite)
http://www.wormatlas.org/neuronalwiring.html

3) This neuron listing (in which they acknowledge that the worm connectome is undergoing revision)

http://www.wormatlas.org/neurons/Individual%20Neurons/Neuronframeset.html
2) the male neuron list :
> http://worms.aecom.yu.edu/jarrelletalSOM.html
>http://worms.aecom.yu.edu/

3) the hermaphrodite neuron list? (supposedly the one at http://www.wormatlas.org/neuronalwiring.html, but they say in other page that it should be 10 or 15 neurons less)

**Comment by imichaelnorris** [comment]
@sgomezr I'm using number 1, the neuronal wiring.  The dataset I'm using on that page is "2.3 Neuron Connections to Sensory Organs and Body Muscles."

What do you mean when you say separate neurons by organism?  Do you mean separate between the hermaphrodite and the male?  There was no separation as far as I know.  There's just a bunch of edges, so if it's a mixture of male and hermaphrodite then I don't know which edge came from where.

Please let me know if there's better dataset that I could use.  I'm using this one because it was used in the Pavlovic, but am not attached to it.

**Comment by sgomezr** [comment]
Yeah, that;s  the problem They're trying to separate them by sex right now. It's a problem because they did separate some cell lists by sex, so I doubt that the lists are comparable.

'Relatedness' ... do you think it refers to whether or not they belong to the same lineage in cell division and migration? (I think I've seen the lineage trees for each neuron somewhere in wormAtlas). Or would it be how far away they're from each other? Now that I think about it... it's most likely the first one. Yeah, in that case, @indivaux 's second comment summarizes well what you can do with that. Sounds cool :)

**Comment by indivaux** [comment]
Yep, in section 2.4 it says "[that ]for each cell, we count the number of cell divisions from the common ancestor where each division is defined as a single unit in length."  So they actually do use the cell lineages to calculate the relatedness metric. Super cool, right?
